rotkehlchen/chain/ethereum/modules/aave/v2/decoder.py

```diff
@@ -25,3 +46,99 @@ def __init__(
             pool_address=POOL_ADDRESS,
             eth_gateways=ETH_GATEWAYS,
         )
+
+    def _decode_staking_events(self, context: DecoderContext) -> DecodingOutput:
```

Comment:
As I said in the previous review and in the standup put this in its own decoder. Makes no sense to mix here.